### PR TITLE
Uploads websocket messages to mongodb

### PR DESCRIPTION
**Pull request by zjayee**
# Description
Uploads websocket messages to mongodb when recieved in backend. 

## Type of change
New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Manual tests

### CODE REVIEW

**Comment by zjayee** [comment]
Should be merged after pr#18

**Comment by 3milyfz** [comment]
Good work! 
- The commit messages are clear 👍
- I also manually tested your branch locally and messages are being stored as expected.

**Comment by zjayee** [comment]
@3milyfz fixed merge conflict

**Review by 3milyfz — APPROVED** [review]

File: src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java [hunk 1]
```diff
@@ -13,22 +17,31 @@
...
     @MessageMapping("/message-employee/{sessionId}")
     @SendTo("/topic/message-customer/{sessionId}")
     public String sendMessageEmployee(@DestinationVariable String sessionId, String message) throws Exception {
+        sendResponseToDatabase(message, sessionId, Sender.EMPLOYEE);
         HashMap<String, String> response = messageInteractor.generateResponse(message);
         return gson.toJson(response);
     }
 
 
+    private void sendResponseToDatabase(String content, String sessionId, Sender sender){
```
Comment (sarinali) [inline src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java:42]: Can you place this method in another file?

Comment (zjayee) [reply src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java]: done moved to MessageDataAccess class

**Pull request merged**